conans/client/installer.py

```diff
@@ -474,3 +482,17 @@ def _call_package_info(conanfile, package_folder):
                     conanfile.build_folder = None
                     conanfile.install_folder = None
                     conanfile.package_info()
+                    if hasattr(conanfile, '_cpp_info_layout_file'):
```

Comment:
We now that node.binary==EDITABLE already. This is not necessary, and hardcoded anyway. So better use a convention over the layout filename.

Reply:
Great suggestion. I plan this feature from the beginning thinking on writing the wrapper around the ConanFile... and that ideas is stuck on my head.